README.md

```diff
@@ -198,9 +205,10 @@ Then, run tests:
 npm test
 ```
 
+### Research
+Our research paper, "Generating GraphQL-Wrappers for REST(-like) APIs", can be found [here](https://arxiv.org/abs/1809.08319). The paper describes the challenges of building OASGraph and an experiment in which we evaluated OASGraph against 959 publicly available OAS, provided by [APIs.guru](https://apis.guru/), and successfully created GraphQL interfaces for 89.5% of them.
 
-### APIs.guru
-OASGraph can be applied to all OAS contained in [APIs.guru OpenAPI repository](https://github.com/APIs-guru/openapi-directory). Load APIs.guru specifications into the `/tmp` folder:
+To run the experiment, load APIs.guru specifications, found [here](https://github.com/APIs-guru/openapi-directory), into the `/tmp` folder:
```

Comment:
I think there are still scripts for doing this, namely `load_apis_guru.js` and `eval_apis_guru.js` in `test/evaluation`. Should we reference to them here?

Reply:
Yeah, that sounds good to me! However, I'm not very familiar with the scripts. Perhaps you can fill me in?

Reply:
The first one populates the `/tmp` folder with the API specifications from APIs.guru. The second runs OASGraph on all of them.

I am not sure if these work properly, though. Maybe you could test them?

Reply:
Oh, I was a little bit confused about what you were saying. The scripts are already mentioned in the README...

Reply:
I tested them and they seem to work

Reply:
Cool!